### PR TITLE
Feature/run openvpn on mobile android

### LINKED ISSUE #364: Create top level mobile package for exposing core apis to mobile platforms
*As a* 3d party ios/Android developer
*I would like to* have a running Tequilapi http server on mobile

AC:
- [x] Create package `mobile/mysterium` with Node entrypoint

Problems:
- [x] Make customised Bootstrap() for `mobile` and `cmd`
- [x] Clear unneeded configs (config args, openvpn args, etc.)
- [x] Don't need to run config checks on `mobile`, just `cmd`

Later:
- have a myst embedded core library with clear api (connect, disconnect, identity management) for my custom app.

### PR DESCRIPTION
**Pull request by tadovas**
Description says it all.
Closes #364

### CODE REVIEW

File: services/openvpn/config_validator.go [hunk 1]
```diff
@@ -121,6 +123,24 @@ func validTLSPresharedKey(config *VPNConfig) error {
 	if len(key) != 512 {
 		return errors.New("invalid key length")
 	}
+
+	var buff = &bytes.Buffer{}
+	fmt.Fprintln(buff, "-----BEGIN OpenVPN Static key V1-----")
+	left := key
+	for {
```
Comment (soffokl) [inline services/openvpn/config_validator.go]: can't we do it like this?

```golang
for left := key; len(left) > 64; left = left[64:] {
   fmt.Fprintln(buff, left[0:64])
}
fmt.Fprintln(buff, left)
```

File: mobile/mysterium/entrypoint.go [hunk 2]
```diff
@@ -20,26 +20,34 @@ package mysterium
 import (
 	"path/filepath"
 
-	log "github.com/cihub/seelog"
+	"github.com/mysteriumnetwork/node/services/openvpn/session"
+
```
[the remark this answers is not in the document]
Comment (tadovas) [reply mobile/mysterium/entrypoint.go]: Some interesting issues with IDE and gofmt. Fixed

File: services/openvpn/config_validator.go (hunk 1 above)
Comment (tadovas) [reply services/openvpn/config_validator.go]: Yes we can. Also we can write additional test for this code part. Done

File: mobile/mysterium/entrypoint.go [hunk 3]
```diff
@@ -20,26 +20,31 @@ package mysterium
...
-	service_noop "github.com/mysteriumnetwork/node/services/noop"
+	"github.com/mysteriumnetwork/node/services/openvpn"
+	"github.com/mysteriumnetwork/node/services/openvpn/session"
 )
 
 // MobileNode represents node object tuned for mobile devices
 type MobileNode struct {
-	di cmd.Dependencies
+	di       cmd.Dependencies
+	identity identity.Identity
```
[the remark this answers is not in the document]
Comment (tadovas) [reply mobile/mysterium/entrypoint.go]: Nope. Leftover

File: mobile/mysterium/entrypoint.go [hunk 4]
```diff
@@ -138,16 +143,18 @@ func (mobNode *MobileNode) WaitUntilDies() error {
...
 // BinaryPath returns noop binary path
-func (noOpenvpnYet) BinaryPath() string {
-	return "no openvpn binary available on mobile"
+func (embbededLibCheck) BinaryPath() string {
+	return "mobile uses embedded openvpn lib"
 }
 
-var _ node.Openvpn = noOpenvpnYet{}
+// check if our struct satisfies Openvpn interface expected by node options
+var _ node.Openvpn = embbededLibCheck{}
```
[the remark this answers is not in the document]
Comment (tadovas) [reply mobile/mysterium/entrypoint.go]: Renamed everywhere

**Pull request merged**